catalog-rest-service/src/main/resources/json/schema/operations/pipelines/airflowPipeline.json

```diff
@@ -46,9 +26,113 @@
             "databaseServiceMetadataPipeline",
             "databaseServiceQueryUsagePipeline"
           ]
+        },
+        "sourceConfig": {
+          "description": "A connection to a dashboard service, database service or messaging service.",
+          "$ref": "../../entity/services/serviceConnection.json#/definitions/serviceConnection"
+        },
+        "sinkConfig": {
+          // let's do the same approach as with the source, currently just with the metadata-rest
         }
       },
       "additionalProperties": false
+    },
+    "airflowConfig": {
+      "description": "Properties to configure the Airflow pipeline that will run the workflow.",
+      "type": "object",
+      "javaType": "org.openmetadata.catalog.operations.pipeline.AirflowConfig",
+      "properties": {
+        "forceDeploy": {
+          "description": "Deploy the pipeline by overwriting existing pipeline with the same name.",
+          "type": "boolean",
+          "default": "true"
+        },
+        "pausePipeline": {
+          "description": "pause the pipeline from running once the deploy is finished successfully.",
+          "type": "boolean",
+          "default": "false"
+        },
+        "concurrency": {
+          "description": "Concurrency of the Pipeline.",
+          "type": "integer",
+          "default": 1
+        },
+        "startDate": {
+          "description": "Start date of the pipeline.",
+          "$ref": "../../type/basic.json#/definitions/date"
```

Comment:
For startDate and endDate we could use `timestamp` to make sure we properly have UTC at conversion https://github.com/open-metadata/OpenMetadata/issues/3655